playbooks/x-portals-run-shell-command.yml

```diff
@@ -0,0 +1,47 @@
+# This playbook expects set `shell_commands` list of commands.
+# E.g. in a file `my-vars/shell-commands.yml` with content:
+#
+# shell_commands:
+#   - "echo abc > abc.txt"
+#   - "docker exec <...>"
```

Comment:
Can we wrap these in "```" in order to make it extra clear that this is the example content of the file? I managed to not understand that, somehow. :/

Also, do we need to quotes? I forgot to add them and it was fine.
This if the file that worked for me:
```
shell_commands:
  - cd /home/user/skynet-webportal && nohup ./dc logs -f pinner>logs/pinner.log &
  - curl -L -X POST http://10.10.10.130:4000/sweep
 ```